webui/ec2/EC2.py

```diff
@@ -130,3 +145,62 @@ def create_user(username):
     u = User(user.user_name, user.user_id, user.create_date,
              [AccessKey(key.id, key.status, key.create_date, key.secret)])
     return u
+
+
+def list_instances(region='eu-central-1'):
+    ec2 = boto3.resource('ec2', aws_access_key_id=provider_conf.EC2['key'],
```

Comment:
you doing this resource init in almost every function , lack of python knowledge stops me from give you certain suggestions in python but from theoretical OOP perspective I would move this resource in some static field/class and will init once for whole app or at least create ConnectionFactory class which handle all things related to task "give me valid API client instance"

Reply:
agree. Lets do it in a other PR.